mizar/dp/mizar/workflows/builtins/pods/create.py

```diff
@@ -175,9 +175,6 @@ def run(self):
             "Initing endpoint interface on for {} on host {}".format(self.param.name, spec['hostIP']))
         interfaces = endpoint_opr.init_simple_endpoint_interfaces(
             spec['hostIP'], spec)
-        if not interfaces:
```

Comment:
If interfaces already exist, why do we need to create this again? At the very least, we should have a warn log here.
@phudtran What's your take on this change?

Reply:
Here the key issue is that it raises a permanent issue. In the early stage of the cluster, the operation of init_simple_endpoint_interfaces may failed, it's not because endpoint exists. But since raised permanent issue, there will be no chance to retry.
I have to choice: 1, change from permanent error to temp error for later to retry. Or 2, just remove it and eventually will trigger retry if hit error. Currently I choose 2. If you think I shall choose 1, I can change to that.

Reply:
If this returns None, I think we should handle it in some way. Letting this codepath proceed will cause just cause a permanent error further down, so removing it does nothing here. (Will fail later down with "unknown error" instead. I prefer to know what caused the error)